src/mlpack/methods/reinforcement_learning/environment/mountain_car.hpp

```diff
@@ -90,16 +90,19 @@ class MountainCar
    *
    * @param positionMin Minimum legal position.
    * @param positionMax Maximum legal position.
+   * @param positionGoal Final target position.
    * @param velocityMin Minimum legal velocity.
    * @param velocityMax Maximum legal velocity.
    */
   MountainCar(const double positionMin = -1.2,
               const double positionMax = 0.5,
+              const double positionGoal = 0.45,
```

Comment:
openai uses default value of `goal_position` as 0.5 & `max_position` as 0.6, so I suggest we should be consistent with it.
https://github.com/openai/gym/blob/12e8b763d5dcda4962cbd17887d545f0eec6808a/gym/envs/classic_control/mountain_car.py#L24